training/src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -20,38 +20,75 @@
 
 import java.util.List;
 
+/**
+ * The type Book controller.
+ */
 @RestController
 @RequestMapping(EndPoints.BOOK_BASE_PATH)
 public class BookController {
 
+    /**
+     * The Book repository.
+     */
     @Autowired
     BookRepository bookRepository;
 
 
+    /**
+     * Greetings! with a name
+     *
+     * @param name:  Optional name of who is going to greet (String)
+     * @param model: Contains the data that appears in the view (Model)
+     * @return The name of the view to perform the greeting
+     */
     @GetMapping("/greeting")
     public String greeting(@RequestParam(name = "name", required = false, defaultValue = "World") String name,
                            Model model) {
         model.addAttribute("name", name);
         return "greeting";
     }
 
+    /**
+     * Find all list.
+     *
+     * @return the list
+     */
     @GetMapping
     public List<Book> findAll() {
         return bookRepository.findAll();
     }
 
+    /**
+     * Create book book.
+     *
+     * @param book the book
+     * @return the book
```

Comment:
Wherever an object is declared on java doc, I recommend you to declare it like this: {@link Book}, so when you generate the documentation, you will see a link to this Object :)

Replace it in every case.

Reply:
Did it already! 😀